ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js

```diff
@@ -0,0 +1,268 @@
+/*!
+ * ReadRemaining.js 1.0.0
```

Comment:
But the tag in [here](https://github.com/Aerolab/readremaining.js/tree/0.1.0) is 0.1.0. Which should I use?